docker/Dockerfile.mediamtx

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:20.04
+
+RUN apt update
+RUN apt install -y curl
+
+RUN curl -L https://github.com/bluenviron/mediamtx/releases/download/v1.9.3/mediamtx_v1.9.3_linux_amd64.tar.gz -o /mediamtx.tar.gz
+RUN tar xzvf /mediamtx.tar.gz
```

Comment:
Would it be useful to also include our own default mediamtx.yml in here?

Reply:
Ah ok yeah just for local development with localhost configured for the gateway call? 👍